org.metaborg.meta.lang.template/trans/generation/pp/to-pp.str

```diff
@@ -578,10 +578,10 @@ rules
     PPAlign(ref, target*) -> NoAnnoList(Op("Align", [ <create-term-from-ref> ref, NoAnnoList(List(<map(create-term-from-ref)> target*))]))
   
   create-constraint-term:
-    Align(ref) -> NoAnnoList(Op("Align", [ <create-term-from-ref> ref]))
```

Comment:
Again, isn't this supposed to be Align instead of AlignList? Note that there are still two different constructors.